CHANGELOG.md

```diff
@@ -2,13 +2,25 @@
 
 All notable changes to the COBOL Language Support extension are documented in this file.
 
+## [0.8.3]
+
+#### Added
+- Syntax highlighting for variables and paragraphs in COBOL file enabled
+
+#### Fixes
+- Java automatic check now recognizes all supported JRE/JDK java implementations
+
 ## [0.8.2]
 
+#### Changes
 - Updates to README
 
 ## [0.8.1]
 
-- Automatic check for Java added before launching the LSP server.
+#### Added
+- Automatic check for Java added before launching the LSP server
+
+#### Changed
```

Comment:
Changes vs Changed - please fix in future release

Reply:
Any objection to me fixing it now in main development branch?